### PR TITLE
Fix clear public chat

### LINKED ISSUE #6679: Clear chat should only clear public chat, not private

### PR DESCRIPTION
**Pull request by BobakOftadeh**
Closes #6679.

### CODE REVIEW

File: bigbluebutton-html5/imports/api/group-chat-msg/server/modifiers/clearGroupChatMsg.js [hunk 1]
```diff
@@ -9,21 +9,19 @@ export default function clearGroupChatMsg(meetingId, chatId) {
...
-        sender: {
-          id: PUBLIC_CHAT_SYSTEM_ID,
-          name: '',
-        },
-        message: CHAT_CLEAR_MESSAGE,
-      };
-
-      return addGroupChatMsg(meetingId, PUBLIC_GROUP_CHAT_ID, clearMsg);
-    });
+    GroupChatMsg.remove({ meetingId, chatId });
```
Comment (antobinary) [inline bigbluebutton-html5/imports/api/group-chat-msg/server/modifiers/clearGroupChatMsg.js]: Is the removal of the callback necessary?

Comment (antobinary) [reply bigbluebutton-html5/imports/api/group-chat-msg/server/modifiers/clearGroupChatMsg.js]: It looks like that's the only change :confused:

File: bigbluebutton-html5/imports/api/group-chat-msg/server/modifiers/clearGroupChatMsg.js [hunk 2]
```diff
@@ -21,18 +21,18 @@ export default function clearGroupChatMsg(meetingId, chatId) {
...
-
-      return addGroupChatMsg(meetingId, PUBLIC_GROUP_CHAT_ID, clearMsg);
+      addGroupChatMsg(meetingId, PUBLIC_GROUP_CHAT_ID, clearMsg);
     });
+    return true;
   }
 
   if (meetingId) {
-    return GroupChatMsg.remove({ meetingId }, () => {
+    return GroupChatMsg.remove({ meetingId, chatId: { $eq: 'MAIN-PUBLIC-GROUP-CHAT' } }, () => {
```
Comment (jfsiebel) [inline bigbluebutton-html5/imports/api/group-chat-msg/server/modifiers/clearGroupChatMsg.js]: You should use `PUBLIC_GROUP_CHAT_ID`.

File: bigbluebutton-html5/imports/api/group-chat-msg/server/modifiers/clearGroupChatMsg.js [hunk 3]
```diff
@@ -21,18 +21,18 @@ export default function clearGroupChatMsg(meetingId, chatId) {
...
       Logger.info(`Cleared GroupChatMsg (${meetingId})`);
     });
   }
 
-  return GroupChatMsg.remove({}, () => {
+  return GroupChatMsg.remove({ chatId: { $eq: 'MAIN-PUBLIC-GROUP-CHAT' } }, () => {
```
Comment (jfsiebel) [inline bigbluebutton-html5/imports/api/group-chat-msg/server/modifiers/clearGroupChatMsg.js]: Same as above.

**Pull request merged**